feature/aft/aft_base/otg_tests/aft_prefixcounters/README.md

```diff
@@ -0,0 +1,178 @@
+# AFT-2.1: AFTs Prefix Counters
+
+## Summary
+
+IPv4/IPv6 prefix counters
+
+## Testbed
+
+* atedut_12.binding
```

Comment:
This test only requires 2 ports, please change to use the 2 port testbed:

https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed

Reply:
done